common/static/sass/_tooltips.scss

```diff
@@ -3,17 +3,26 @@
   @extend %ui-depth5;
   position: absolute;
   opacity: 0.0;
-  line-height: 26px !important;
+  line-height: 24px !important;
 
   > .tooltip-content {
     @extend %t-regular;
     line-height: inherit;
     font-size: 12px;
     max-width: 800px;
-    padding: 0 10px;
+    padding: 2px 5px;
     border-radius: 3px;
     background: rgba(0, 0, 0, 0.85);
     color: $white;
+
+    a {
+      color: #66aed0 !important;
```

Comment:
For colors, you should use one of [the color variables](https://github.com/edx/edx-platform/blob/5bb0e908/cms/static/sass/_variables.scss) instead of putting the RGB value.